kuksa-sdk/consumer-rules.pro

```diff
@@ -0,0 +1,2 @@
+-keep class org.eclipse.kuksa.proto.v1.** { *; }
+-keep class com.google.protobuf.** { *; }
```

Comment:
redundant with your branch, but I had to add these rules again, to make sure running the app minified is working in the current state